embed_grader/serapis/templates/serapis/assignment.html

```diff
@@ -144,7 +144,7 @@ <h2 class="page-header">Assignment - {{ assignment.name }}</h2>
         {% if num_team_members == 1 %}
           you are the only person in the team.
         {% else %}
-          there are {{num_team_members}} people in your team.
+          there are {{num_team_members}} people in your team. They are {{team_members_human_readable}}.
```

Comment:
`there are {{num_team_members}} people in your team ({{team_members_human_readable}}.)`

It's just a little bit to say "they are", because the viewer is part of the team